Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -661,6 +661,11 @@ private void SurfaceObserver_OnDataReady(SurfaceData cookedData, bool outputWrit
                     meshObject.Renderer.sharedMaterial = (displayOption == SpatialAwarenessMeshDisplayOptions.Visible) ?
                         VisibleMaterial :
                         OcclusionMaterial;
+
+                    if (PhysicsMaterial != null)
```

Comment:
@davidkline-ms  I think that this code only gets invoked at mesh creation, so we don't have to worry about the reverse transition (i.e. someone setting PhysicsMaterial to null to clear it at runtime) right?